Gemfile

```diff
@@ -40,7 +41,7 @@ group :development, :test do
   gem 'rspec-rails'
   gem 'rubocop'
   gem 'site_prism'
-  gem 'spring'
+  gem 'spring', '1.3.6'
```

Comment:
We were getting an error that one dependency had loaded 1.3.6 and another part of the codebase needed 1.3.4 (I think was the number). Went with forcing it to the higher number.